backend/README.md

```diff
@@ -1,100 +1,158 @@
 # Hanko backend
 
-## Usage
+Hanko backend provides an HTTP API to build a modern login and registration experience for your users. Its core features are an API for passkeys (WebAuthn), passwords, and passcodes, as well as JWT management.
 
-### Serving api
-#### Public
-```shell
-go run main.go serve public
+Hanko backend can be used on its own or in combination with [hanko-js](../hanko-js), a powerful frontend library that contains polished and customizable UI flows for password-based and passwordless user authentication that can be easily integrated into any web app with as little as two lines of code.
+
+## API features
+
+- Passkeys (WebAuthn)
+- Passcodes
+- Passwords
+- Email verification
+- JWT management
+
+## Upcoming features
+
+- Exponential backoff for password attempts and passcode email sending
+- More tests and code documentation
+
+## Basic usage
+
+The easiest way to start Hanko backend service is through docker. But before we can do that, we need to create a config file.
+
+> **Note** If you just want to jump right into the experience of passkeys and passcodes, head over to the [quickstart guide](../README.md#Quickstart).
```

Comment:
> [...] the [quickstart guide](../README.md#Quickstart).

There is no heading (and section/guide) with that ID it just takes me to the top of the readme.

Reply:
Any idea how this can be fixed without using the absolute url?

Reply:
Wel, you could try using the "correct" ID for a start, so maybe:

```suggestion
> **Note** If you just want to jump right into the experience of passkeys and passcodes, head over to the [quickstart guide](../README.md#quickstart).
```

Reply:
Nope, does not work. I'll just try something else real quick.